pandas/core/generic.py

```diff
@@ -2521,69 +2521,102 @@ def to_xarray(self):
-            columns
-        longtable : boolean, default will be read from the pandas config module
-            Default: False.
-            Use a longtable environment instead of tabular. Requires adding
-            a \\usepackage{longtable} to your LaTeX preamble.
-        escape : boolean, default will be read from the pandas config module
+            <https://en.wikibooks.org/wiki/LaTeX/Tables>`__ e.g. 'rcl' for 3
+            columns.
+        longtable : bool, default None
+            When set to None, the value will default from the pandas config
+            module. Use a longtable environment instead of tabular. Requires
+            adding a \usepackage{longtable} to your LaTeX preamble.
+        escape : bool, default will be read from the pandas config module
             Default: True.
             When set to False prevents from escaping latex special
             characters in column names.
         encoding : str, default None
             A string representing the encoding to use in the output file,
             defaults to 'ascii' on Python 2 and 'utf-8' on Python 3.
-        decimal : string, default '.'
+        decimal : str, default '.'
             Character recognized as decimal separator, e.g. ',' in Europe.
-
             .. versionadded:: 0.18.0
-
-        multicolumn : boolean, default True
+        multicolumn : bool, default True
             Use \multicolumn to enhance MultiIndex columns.
             The default will be read from the config module.
-
             .. versionadded:: 0.20.0
-
         multicolumn_format : str, default 'l'
             The alignment for multicolumns, similar to `column_format`
             The default will be read from the config module.
-
+            .. versionadded:: 0.20.0
+        multirow : bool, default False
+            Use \multirow to enhance MultiIndex rows. Requires adding a
+            \usepackage{multirow} to your LaTeX preamble. Will print
+            centered labels (instead of top-aligned) across the contained
+            rows, separating groups via clines. The default will be read
+            from the pandas config module.
             .. versionadded:: 0.20.0
 
-        multirow : boolean, default False
-            Use \multirow to enhance MultiIndex rows.
-            Requires adding a \\usepackage{multirow} to your LaTeX preamble.
-            Will print centered labels (instead of top-aligned)
-            across the contained rows, separating groups via clines.
-            The default will be read from the pandas config module.
+        Returns
+        -------
+        str or None
+            If buf is None, returns the resulting LateX format as a
+            string. Otherwise returns None.
 
-            .. versionadded:: 0.20.0
-            """
+        See Also
+        --------
+        DataFrame.to_csv : Write a DataFrame to CSV format.
+        DataFrame.to_excel : Write a DataFrame to an Excel file.
```

Comment:
I'd have `to_string` and `to_html` in this case, which to me are conceptually more similar than `to_csv` or `to_excel` (formatting to present vs formatting to export).